src/content/advanced/gitops/bases/index.md

```diff
@@ -0,0 +1,174 @@
+---
+linkTitle: Creating a base template clusters
+title: Creating a base template for workload clusters
+description: How to create a base template to create workload clusters with different configurations.
+weight: 70
+menu:
+  main:
+    parent: advanced-gitops
+    identifier: advanced-gitops-bases
+user_questions:
+  - How can I create an base template for workload clusters in GitOps?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+last_review_date: 2023-01-20
+---
+
+# Add a CAPx (CAPI) Workload Cluster template (cluster App based)
+
+Our CAPx (CAPI provider-specific clusters) are delivered by Giant Swarm as a set of two applications. First one
+is an App with Cluster instance definition, second one is an App with all the default applications a new cluster
+needs in order to run correctly. As such, creating a CAPx cluster means that you need to deliver two correctly
+configured App CRs to the Management Cluster.
+
+Adding definitions can be done on two levels: shared cluster template and version specific template, see
+[create shared template base](#create-shared-template-base)
+and [create versioned base](#create-versioned-base-optional).
+
+
+**IMPORTANT**, CAPx configuration utilizes the
+[App Platform Configuration Levels](/getting-started/app-platform/app-configuration/#levels),
+in the following manner:
+
+- cluster templates provide default configuration via App' `config` field,
+- cluster instances provide custom configuration via App' `userConfig` field, that is overlaid on top of `config`.
+
+See more about this approach [here](https://github.com/giantswarm/rfc/tree/main/merging-configmaps-gitops).
+
+## Choose bases
+
+In order to avoid code duplication, it is advised to utilize the
+[bases and overlays](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#bases-and-overlays)
+of Kustomize in order to configure the cluster.
```

Comment:
```suggestion
In order to avoid code duplication, it is advised to utilize the [bases and overlays](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#bases-and-overlays) of Kustomize in order to configure the cluster.
```
I think you idea breaks the lines at 80 characters which does not work with docs md well